### PR TITLE
Bump rancher-csp-adapter version

### PR DESCRIPTION
**Pull request by steffantucker**
Bumps the version of CSP adapter to 2.0.0

### CODE REVIEW

**Comment by steffantucker** [comment]
Don't merge until rancher/charts#2164 has been merged

**Review by maxsokolovsky — APPROVED** [review]

**Review by MbolotSuse — APPROVED** [review]

**Pull request merged**
